docs/vm/create-vm.md

```diff
@@ -99,6 +101,17 @@ It is also possible to connect VMs using additional networks with Harvester's bu
 
 In bridge VLAN, virtual machines are connected to the host network through a linux `bridge`. The network IPv4 address is delegated to the virtual machine via DHCPv4. The virtual machine should be configured to use DHCP to acquire IPv4 addresses.
 
+## Node Scheduling
+`Node Scheduling` allows you to constrain which nodes your VMs can be scheduled on based on node labels.
+
+See the [Kubernetes Node Affinity Documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) for more details.
+
+## VM Scheduling
+
+`VM Scheduling` allows you to constrain which nodes your VMs can be scheduled on based on the labels of workloads (VMs and Pods) already running on that node, instead of the node labels.
```

Comment:
```suggestion
`VM Scheduling` allows you to constrain which nodes your VMs can be scheduled on based on the labels of workloads (VMs and Pods) already running on these nodes, instead of the node labels.
```